src/components/network/network-map.js

```diff
@@ -505,6 +506,7 @@ NetworkMap.defaultProps = {
     filteredNominalVoltages: null,
     labelsZoomThreshold: 9,
     arrowsZoomThreshold: 7,
+    tooltipZoomThreshold: 5,
```

Comment:
We tried different values instead of 5 and 7 seems better, can you make the change ?

Reply:
It was validated by the PO